_posts/2024/2024-02-06-our-process.md

```diff
@@ -0,0 +1,453 @@
+data scientists,
+and machine learning specialists staring into the void and hoping it blinks first.
+And while most of us are in Toronto,
+a substantial minority live and/or work remotely.
+This meeting is where the VP of Engineering updates us all
+on things we might otherwise miss
+(both organizationally and technically).
+
+-   Summary: Biweekly team sync to discuss company and/or engineering news
+-   Key Questions:
+    -   What's going on elsewhere in the company that might affect us?
+    -   What's coming up that we ought to know about?
+-   Frequency: biweekly
+-   Responsible: VP Engineering
+-   Contributors: entire engineering team plus invited guests
+-   Products:
+    -   Heightened awareness of recent and upcoming changes
+
+### Engineering Design Reviews
+
+One of the things that has come out of our end-of-quarter reviews
+is a greater emphasis on presenting designs for new features or applications to the whole team
+before diving into implementation.
+The goal here is *not* to create a pile of UML diagrams that no one will ever look at again,
+but rather to explain what the problem actually is,
+what factors a solution must take into account,
+what alternatives there are,
+and what we're *not* going to do.
+Some designs go through two or three overhauls before work starts;
+in almost every case,
+checking our thinking like this has saved us pain downstream.
+
+-   Summary: Weekly sync to present and discuss technical proposals
+-   Key Questions:
+    -   What problem is the design solving?
+    -   For who?
+    -   How are they solving the problem today?
+    -   Why isn't that good enough?
+    -   What is out of scope?
+-   Frequency: weekly
+-   Responsible: VP Engineering
+-   Contributors: entire engineering team
+-   Products:
+    -   Designs for new features or applications
+    -   Lateral awareness of who's doing what and why
+    -   Buy-in from the team (or at least a chance to air our differences)
+
+> A recent change is to update the design doc a few weeks after work starts
+> to explain where and why reality has diverged from the plan.
+> Doing this extends the useful life of that document,
+> but more importantly,
+> it's a chance to reflect on what the code is teaching us
+> and share that with other people
+> so that our next design(s) can be better.
+
+### Weekly Friday Engineering Demos
+
+One of our more experienced team members instituted Friday afternoon demo sessions last year,
+and we have all become believers.
+Some demos show off a new feature;
+others walk the audience through a bug that has been fixed
+or introduce the team to useful third-party tools.
+Demos runs anywhere from a few minutes to half an hour,
+with most being near the low end of that range.
+It's a fun way to wrap up the week and remind ourselves that we're actually making progress.
+
+-   Summary: Weekly sync for team to share ongoing work and discoveries
+-   Key Questions:
+    -   It does *what*??
```

Comment:
Thinking of your "how to demo" training, perhaps you can add the three questions you had in there: What problem is it solving, for who, and how does it work (paraphrasing here as I can't recall the exact content)

Reply:
done